src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -833,33 +828,39 @@ public void shutdown() throws InterruptedException {
     @Getter
     @Data
     static class CurrentProcessingFileInformation {
-        @JsonProperty(PERSISTED_CURRENT_PROCESSING_FILE_NAME)
-        private String fileName;
         @JsonProperty(PERSISTED_CURRENT_PROCESSING_FILE_START_POSITION)
         private long startPosition;
         @JsonProperty(PERSISTED_CURRENT_PROCESSING_FILE_LAST_MODIFIED_TIME)
         private long lastModifiedTime;
+        @JsonProperty(PERSISTED_CURRENT_PROCESSING_FILE_HASH)
+        private String fileHash;
 
         public Map<String, Object> convertToMapOfObjects() {
             Map<String, Object> currentProcessingFileInformationMap = new HashMap<>();
-            currentProcessingFileInformationMap.put(PERSISTED_CURRENT_PROCESSING_FILE_NAME, fileName);
```

Comment:
Lets avoid deleting this key and I think it is ok to keep the fileName here. Just tag it as deprecated and signal that it should be removed. I just want to avoid people upgrading and then downgrading from breaking

Reply:
Agree.